Source/Widgets/Viewer/viewerDragDropMixin.js

```diff
@@ -150,6 +152,20 @@ define([
             },
 
             /**
+             * Gets or sets a value indicating if the camera should fly to the data source after it is loaded.
+             * @memberof viewerDragDropMixin.prototype
+             * @type {Boolean}
+             */
+            flyToOnDrop : {
+                get : function() {
+                    return flyToOnDrop;
+                },
+                set : function(value) {
+                    flyToOnDrop = value;
```

Comment:
Is there a reason we do this, as opposed to having a simple property, or knockout?  Just curious.